src/EditorFeatures/Test/CodeGeneration/CodeGenerationTests.VisualBasic.cs

```diff
@@ -710,31 +835,48 @@ End Class
             public async Task AddStatementsToSub()
             {
                 var input = "Class C\n [|Public Sub M\n Console.WriteLine(1)\n End Sub|]\n End Class";
-                var expected = "Class C\n Public Sub M\n Console.WriteLine(1)\n $$\n End Sub\n End Class";
+                var expected = @"Class C
+ Public Sub M
+ Console.WriteLine(1)
+$$ End Sub
+ End Class";
                 await TestAddStatementsAsync(input, expected, "Console.WriteLine(2)");
             }
 
             [Fact, Trait(Traits.Feature, Traits.Features.CodeGeneration)]
             public async Task AddStatementsToOperator()
             {
                 var input = "Class C\n [|Shared Operator +(arg As C) As C\n Return arg\n End Operator|]\n End Class";
-                var expected = "Class C\n Shared Operator +(arg As C) As C\n Return arg\n $$\n End Operator\n End Class";
+                var expected = @"Class C
+ Shared Operator +(arg As C) As C
+ Return arg
+$$ End Operator
+ End Class";
                 await TestAddStatementsAsync(input, expected, "Return Nothing");
             }
 
             [Fact, Trait(Traits.Feature, Traits.Features.CodeGeneration)]
             public async Task AddStatementsToPropertySetter()
             {
                 var input = "Imports System\n Class C\n WriteOnly Property P As String\n [|Set\n End Set|]\n End Property\n End Class";
-                var expected = "Imports System\n Class C\n WriteOnly Property P As String\n Set\n $$\n End Set\n End Property\n End Class";
+                var expected = @"Imports System
+ Class C
+ WriteOnly Property P As String
+ Set
+$$ End Set
+ End Property
+ End Class";
                 await TestAddStatementsAsync(input, expected, "Console.WriteLine(\"Setting the value\"");
             }
 
             [Fact, Trait(Traits.Feature, Traits.Features.CodeGeneration)]
             public async Task AddParametersToMethod()
             {
                 var input = "Class C\n Public [|Sub M()\n End Sub|]\n End Class";
-                var expected = "Class C\n Public Sub M(num As Integer, Optional text As String = \"Hello!\", Optional floating As Single = 0.5)\n End Sub\n End Class";
+                var expected = @"Class C
+ Public Sub M(numAs Integer, OptionaltextAs String = ""Hello!"",OptionalfloatingAs Single = 0.5)
```

Comment:
I don't think the codegenerator does any formatting.  You'd need to pass hte node to a formatter to do that sort of thing.